second-gen/packages/swc/components/progress-circle/stories/progress-circle.stories.ts

```diff
@@ -139,7 +139,7 @@ export const Indeterminate: Story = {
 export const StaticWhite: Story = {
     render: () => html`
         <div
-            style="background-color: rgba(0,0,0,0.4); padding: 24px; display: flex; gap: 24px; align-items: center;"
+            style="background: linear-gradient(45deg, rgb(64 0 22), rgb(14 24 67)); padding: 24px; display: flex; gap: 24px; align-items: center;"
```

Comment:
Added our pretty gradients here for static white/black which match how design demos these styles. They're meant to be on visually busy backgrounds like gradients or images so that drives home that guidance in Storybook as well.